docs/pages/database-access/reference/cli.mdx

```diff
@@ -157,6 +157,66 @@ $ tctl auth sign --format=db --host=host1,localhost,127.0.0.1 --out=db --ttl=219
 | `--out` | Name prefix for output files. |
 | `--ttl` | Certificate validity period. |
 
+<Details title="Setting up RBAC for signing database certificates">
+
+The `tctl` user must have permissions to impersonate the Teleport Database
+Service user, `Db`, in order to generate a signed database certificate. To add
+these impersonation privileges to your Teleport user, run the following
+commands.
+
+First, define a role that can impersonate the `Db` user. Add the following
+content to a file called `db-impersonator.yaml`:
+
+```yaml
+kind: role
+version: v5
+metadata:
+  name: db-impersonator
+spec:
+  options:
+  allow:
+    impersonate:
+      users: ['Db']
```

Comment:
I'm not certain whether `users: ['Db']` is actually required. Have you tried without?

Reply:
@ptgott @zmb3 IIRC right now this is required but to better enable Teleport Discover flow @marcoandredinis and I are going to introduce another resource kind "DatabaseCert" and update our preset editor role with this permission. Then this no longer be required and we'll be able to remove this extra step from the docs also as it would just work out of the box.

Reply:
Are you sure? I think Machine ID changes made it so that you can specify roles or users but not both.

@timothyb89?

Reply:
That is true, but it's only useful if you set specific flags when requesting certs later (specifically the `UseRoleRequests` and `RoleRequests` flags on `UserCertsRequest`). `tctl auth sign` doesn't support this so I think it's still functionally true that `users` are required in most cases.

Reply:
As of v9.3.8 (my Teleport Cloud account), I needed both the `users` and `roles` fields to be able to run the `tctl auth sign` commands here.

Reply:
For reference, here's the PR that adds the DatabaseCertificate as a resource kind
https://github.com/gravitational/teleport/pull/14526

Reply:
@zmb3 Just double-checking if I should change this to remove the `users` field.